.ci.yaml

```diff
@@ -2345,6 +2345,18 @@ targets:
       task_name: android_lifecycles_test
     scheduler: luci
 
+  - name: Linux_android openpay_benchmarks__scroll_perf
```

Comment:
We need a platform section for `linux_samsung_a02` similar to `s10`: https://github.com/flutter/flutter/blob/master/.ci.yaml#L42, but with `device_type: "SM-A025V"`.

And then rename the builder name to `Linux_samsung_a02 openpay_benchmarks__scroll_perf`.

Reply:
Done!